### ISSUE TITLE
Custom SVG icons for Point Layer don't preview

### ISSUE THREAD

**Issue by radumas** [tomchadwin/qgis2web#95]
Using `Ubuntu 14.04`, `QGIS 2.10.1`, `Python 2.7.6` and `qgis2web` I get the following error when I hit preview for a Leaflet map with a point layer with a custom SVG for icons. I have tried various combinations of layers and visibility, the error stops when I remove displaying the layer with SVG icons.

```
Traceback (most recent call last):
  File "/home/user/.qgis2/python/plugins/qgis2web/maindialog.py", line 65, in changeFormat
    self.previewMap()
  File "/home/user/.qgis2/python/plugins/qgis2web/maindialog.py", line 99, in previewMap
    MainDialog.previewLeaflet(self)
  File "/home/user/.qgis2/python/plugins/qgis2web/maindialog.py", line 229, in previewLeaflet
    previewFile = writeLeaflet(self.iface, utils.tempFolder(), 500, 700, 1, layers, visible, "", cluster, "", "", "", labels, 0, 0, json, params, popup)
  File "/home/user/.qgis2/python/plugins/qgis2web/leafletWriter.py", line 239, in writeLeaflet
    borderStyle = symbol.symbolLayer(0).outlineStyle()
AttributeError: 'QgsSvgMarkerSymbolLayerV2' object has no attribute 'outlineStyle'
```

**Update:**
It doesn't display in either Leaflet or OL3, but the error message only pops up in Leaflet.

**Comment by tomchadwin** [comment]
Not sure the cause of the OL3 failure, but the Leaflet one looks easy to fix.

**Comment by tomchadwin** [comment]
Confirmed - looking into it now.

**Comment by tomchadwin** [comment]
SVG markers are not supported in Leaflet output. I've just made a commit to catch these errors, and output a point marker instead. Can you test Leaflet preview and let me know if the crash is gone?

**Comment by tomchadwin** [comment]
Also confirmed that the icons aren't showing in OL3 - no style comes through at all for me. I've committed a fix for the path to the SVG file, but it's still not working. Can anyone figure out why?

**Comment by tomchadwin** [comment]
Correction: I was using QGIS SVGs, and they seem not to work - I don't know why. Using an alternative SVG works. Have asked about this on qgis-dev: http://osgeo-org.1560.x6.nabble.com/QGIS-SVGs-won-t-work-in-OpenLayers3-td5239335.html.

**Comment by tomchadwin** [comment]
I've now added SVG support to Leaflet in master. That seems to work with the QGIS SVG markers which don't work in OL3. Still no idea on that issue.

**Comment by tomchadwin** [comment]
Released as 1.10.0-FOSS4GUK2016 experimental and pushed to QGIS plugins repo.
